eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -348,18 +349,29 @@ function CreateWorkItem($title, $type, $iteration, $area, $fields, $assignedTo,
 
   $workItem = Invoke-AzBoardsCmd "work-item create" $parameters $outputCommand
 
-  if ($parentId) {
-    $parameters = $ReleaseDevOpsCommonParameters
-    $parameters += "--id", $workItem.id
-    $parameters += "--relation-type", "parent"
-    $parameters += "--target-id", $parentId
-
-    Invoke-AzBoardsCmd "work-item relation add" $parameters $outputCommand | Out-Null
+  if ($parentId)
```

Comment:
Should these 2 be mutually exclusive?

Reply:
Package work item will have product work item as `parent` and release plan work item as `related`